src/pandas_profiling/profile_report.py

```diff
@@ -90,6 +91,7 @@ def __init__(
         self._json = None
         self._typeset = None
         self._summarizer = None
+        self._infer_dtypes = None
```

Comment:
The line above includes infer_dtypes in the settings (`config.set_kwargs(kwargs)`). This line and 151 can be removed

Reply:
Sorry my bad! I rechecked. Initially I used this method, finally I tried using config.
Yes, we can remove lines that has `infer_dypes` in `profile_report.py` file as the kwarg is stored using `config.set_kwargs(kwargs)`.